Index.html

```diff
@@ -0,0 +1,185 @@
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>fCC Technical Documentation Page</title>
+	<link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@500&display=swap" rel="stylesheet">
+</head>
+<body>
+  <nav id="navbar">
+    <header id="navbar-title">Basic CSS Documentation</header>
+    <a class="nav-link" href="#Introduction">Introduction</a>
+    <a class="nav-link" href="#what_is_css">What is CSS?</a>
+    <a class="nav-link" href="#Anatomy_of_a_CSS_ruleset">Anatomy of a CSS ruleset</a>
+    <a class="nav-link" href="#Selecting_Multiple_Elements">Selecting Multiple Elements</a>
+    <a class="nav-link" href="#Different_Types_of_Selectors">Different types of selectors</a>
+    <a class="nav-link" href="#CSS:_all_about_boxes">CSS: all about boxes</a>
+    <a class="nav-link" href="#Reference">Reference</a>
+  </nav>
+	<main id="main-doc">
+	  <section class="main-section" id="Introduction">
+      <header>Introduction</header>
+      <article>
+        <p>CSS (Cascading Style Sheets) is the code that styles web content. CSS basics walks through what you need to get started. </p>
+        <p>Answering questions like: How do I make text red? How do I make content display at a certain location in the (webpage) layout? How do I decorate my webpage with background images and colors?</p>
+      </article>		
+	  </section>
+		
+    <section class="main-section" id="what_is_css">
+      <header>what is css?</header>
```

Comment:
- Each .main-section should have an id that matches the text of its first child, having any spaces in the child's text replaced with underscores (_) for the ids.
  - Here you have a question mark at the end of your text which you don't have in your ID add it to solve the issue.

Reply:
@ahzamir I implemented the feedback kindly take a look.